src/doc/documentation-guide.md

```diff
@@ -94,6 +95,72 @@ that concatenates the various `README.md` files we have.
 
 # Stylistic Conventions
 
+## General Considerations
+
+Documentation is most effective when it is precise, concise and uniform.
+Documentation is not a novel, or a blog post. It should not be fun to read.
+Documentation should be normative, and as such be an especially unsurprising
```

Comment:
There are a lot of things that are great about better doc standards but I don't share the opinions in this "general considerations" section.  Concretely, my differing opinions:

 * Documentation can be boring but it does not have to be.
 * Documentation can be both good and fun.  Good is a requirement, fun is a nice-to-have.
 * 11 lines on being concise is too much
 * It feels like more of a blog post than documentation ;)  Or rather: too much opining for the guidelines.
 * It is a backhanded commentary on my latest doc merge without @-tagging me, but expressed as a PR instead of an issue to discuss.
 * A lost opportunity to use the word "concision", which sounds like scissors and also rhymes with "precision"!  A travesty!